doc/interop-test-descriptions.md

```diff
@@ -452,10 +466,9 @@ Asserts:
 
 ### unimplemented_method
 
-Status: Not yet implementable
+Status: Ready for implementation. Blocking beta.
```

Comment:
This is not implementable because UnimplementedCall is not defined. It seems we should have a second service in test.proto that has the unimplemented call.